# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1455214216 18000
#      Thu Feb 11 13:10:16 2016 -0500
# Node ID 5ddc9b61a0c4053c3b53079078862d261b2e6a8c
# Parent  7be239ae42c81eeaf61b8ec67cb3982249eff112
* Makefile.am (BUILT_SOURCES): Add octave-config.h to the list.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -140,16 +140,17 @@ endif
 if AMCOND_HAVE_ICON_TOOLS
 icons-dist-hook:
 else
 icons-dist-hook:
 	@echo "Packaging distribution requires icotool and rsvg-convert." ; exit 1;
 endif
 
 BUILT_SOURCES += \
+  octave-config.h \
   run-octave \
   $(DIRSTAMP_FILES)
 
 if AMCOND_HAVE_BROKEN_STL_ALGO_H
   BUILT_SOURCES += bits/stl_algo.h
 else
   BUILT_SOURCES += nonexistent-file
 endif
